packages/mobile/src/screens/collection-screen/CollectionHeader.tsx

```diff
@@ -207,7 +209,7 @@ const OfflineCollectionHeader = (props: OfflineCollectionHeaderProps) => {
       </View>
       <View style={styles.headerRight}>
         <Switch
-          value={isMarkedForDownload}
+          defaultValue={isMarkedForDownload}
```

Comment:
updated this to defaultValue so that the rapid presses of the toggle would display the proper value instead of flipping back to the real value and looking buggy